jupyter_server/services/contents/manager.py

```diff
@@ -75,10 +75,19 @@ def emit(self, data):
 
     root_dir = Unicode("/", config=True)
 
+    preferred_dir = Unicode(
+        "/",
+        config=True,
+        help=_i18n(
+            "Preferred starting directory to use for notebooks as a path local to `root_dir`."
```

Comment:
Per our previous conversation, I find "local" to be somewhat ambiguous and wonder if "as a subdirectory of `root_dir`" or "relative to `root_dir`" might be more clear.

If a user would like to set `preferred_dir` to be the same as `root_dir`, would they set `preferred_dir = "."` or set `preferred_dir = None`?  This seems to argue for its default value to be `"."` since `"/"` is absolute.